src/pages/LoginPage.tsx

```diff
@@ -3,20 +3,48 @@ import Form from '../components/Form';
 import EmailInput from '../components/EmailInput';
 import PasswordInput from '../components/PasswordInput';
 import { Link as RouterLink } from 'react-router-dom';
-import { Link } from '@mui/material';
+import { Link, Typography } from '@mui/material';
 
 const LoginPage: React.FC = () => {
   const [email, setEmail] = useState<string>('');
   const [password, setPassword] = useState<string>('');
+  const [loginError, setLoginError] = useState<string | null>(null);
 
-  const handleLogin = (event: React.FormEvent<HTMLFormElement>) => {
+  const handleLogin = async (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
 
-    console.log('Email: ', email);
-    console.log('Password: ', password);
+    const requestBody = {
+      email,
+      password
+    };
 
-    setEmail('');
-    setPassword('');
+    try {
+      const response = await fetch('/auth/login', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(requestBody)
+      });
```

Comment:
Might be good to start moving functions for making API calls into a separate file, like an API utility or something and then just call them where you need them.

Also, the same comments from registration-api PR apply here.